src/textual/widgets/_list_view.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import annotations
+
+from textual import events
+from textual.binding import Binding
+from textual.containers import Vertical
+from textual.geometry import clamp
+from textual.message import Message
+from textual.reactive import reactive
+from textual.widgets._list_item import ListItem
+
+
+class ListView(Vertical, can_focus=True, can_focus_children=False):
+    DEFAULT_CSS = """
+    ListView {
+        scrollbar-size-vertical: 2;
+    }
+    """
+
+    BINDINGS = [
+        Binding("down", "cursor_down", "Down", show=False),
+        Binding("up", "cursor_up", "Up", show=False),
+        Binding("enter", "select_cursor", "Select", show=False),
+    ]
+
+    index = reactive(0, always_update=True)
+
+    def __init__(
+        self,
+        *children: ListItem,
+        initial_index: int | None = 0,
+        name: str | None = None,
+        id: str | None = None,
+        classes: str | None = None,
+    ) -> None:
+        super().__init__(*children, name=name, id=id, classes=classes)
+        self.index = initial_index
+
+    @property
+    def highlighted_child(self) -> ListItem | None:
+        """ListItem | None: The currently highlighted ListItem,
+        or None if nothing is highlighted.
+        """
+        if self.index is None:
+            return None
+        elif 0 <= self.index < len(self.children):
+            return self.children[self.index]
+
+    def validate_index(self, index: int | None) -> int | None:
+        if not self.children or index is None:
+            return None
+        return self._clamp_index(index)
+
+    def _clamp_index(self, index: int) -> int:
+        """Clamp the index to a valid value given the current list of children"""
+        last_index = max(len(self.children) - 1, 0)
+        return clamp(index, 0, last_index)
+
+    def _is_valid_index(self, index: int | None) -> bool:
+        """Return True if the current index is valid given the current list of children"""
+        if index is None:
+            return False
+        return 0 <= index < len(self.children)
```

Comment:
I find something like ↓ nicer. Is it, or should I see a doctor about my preferences?

```suggestion
        return index is not None and 0 <= self.index < len(self.children)
```

Reply:
I prefer to keep the two cases as I find it more readable :)